### PR TITLE
[Ready] [Testmerge] MK II Cargo Rework - Sell harder edition 

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

Removes the default Elstic on most things as a base -Fuck that shit said by everyone
Most cooked/made food now sells,
Silk has been made!
Two new cargo crates one barrel on Loom
The third new crate is 40 sheets(?) of cotten for manufactering 
Silk can be sold for 200 per woven sheet
Wooden barrles now only take 10 wood to be made
Cleans up the wood crafting menu adds spaces.
More mech parts and weaponds can be sold!
Upped the export of mechs so their worth it
Fixed issues with breath implants

##Still working on booze crafting!

## Why It's Good For The Game

More options and gear to sell to CC/NT for the time being well still keeping it harder to get money other then waiting for miners or setting up Factoroi 2d spess men addition 

## Changelog
:cl:
add: six more crates, A barrel, A Loom, 40 cotton sheets, two sets of fruit crates, raw lumber crate
add: All fermi chems, Boozes, Medical, food chems now sell
add: Loads more to sell - Mech gear, Cooking and more!
tweak: Moved around the vaule of some things and removed elastic of most items 
balance: Rebreather implants will now loss vaule, Do to being just metal and glass
balance: lowered how many chems are in lewd chem kegs to be around 150-100 as well as the fancy booze kegs
/:cl:

### CODE REVIEW

File: code/modules/cargo/exports/food_wine.dm [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+	cost = 4
+	unit_name = "cooked protein based food"
+	export_types = list(/obj/item/reagent_containers/food/snacks/meat/cutlet)
+
+/datum/export/food/cooked_meat
+	cost = 8
+	unit_name = "cooked protein based food"
+	export_types = list(/obj/item/reagent_containers/food/snacks/meat/steak)
+
+/datum/export/food/douge
```
Comment (dapnee) [inline code/modules/cargo/exports/food_wine.dm]:
```suggestion
/datum/export/food/dough
```

File: code/modules/cargo/exports/food_wine.dm [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+	cost = 3
+	unit_name = "uncooked food base"
+	export_types = list(/obj/item/reagent_containers/food/snacks/dough, /obj/item/reagent_containers/food/snacks/flatdough)
+
+/datum/export/food/cooked_douge
```
Comment (dapnee) [inline code/modules/cargo/exports/food_wine.dm]:
```suggestion
/datum/export/food/cooked_dough
```

File: code/modules/cargo/packs/materials.dm [hunk 3]
```diff
@@ -168,3 +168,19 @@
...
+	name = "Loom"
+	desc = "A large pre-made loom."
+	cost = 1000
+	contains = list(/obj/structure/loom)
+	crate_name = "loom crate"
+	crate_type = /obj/structure/closet/crate/large
+
+/datum/supply_pack/materials/wooden_barrel
+	name = "Wooden Barrel"
+	desc = "A wooden barrles ready for storge."
```
Comment (dapnee) [inline code/modules/cargo/packs/materials.dm]:
```suggestion
	desc = "Wooden barrels ready for storage."
```

File: code/modules/cargo/packs/organic.dm [hunk 4]
```diff
@@ -229,6 +248,16 @@
 	crate_name = "potted plants crate"
 	crate_type = /obj/structure/closet/crate/hydroponics
 
+/datum/supply_pack/organic/rawcotton
+	name = "Raw Cotton Crate"
+	desc = "Plushies have been on the down in the market, and now do to a flood of raw cotton the price of it is so cheap its a steal! Contains 40 raw cotton sheets."
```
Comment (dapnee) [inline code/modules/cargo/packs/organic.dm]:
```suggestion
	desc = "Plushies have been on the down in the market, and now due to a flood of raw cotton the price of it is so cheap, its a steal! Contains 40 raw cotton sheets."
```

File: code/modules/cargo/exports/organs_robotics.dm [hunk 5]
```diff
@@ -110,6 +111,17 @@
 	export_types = list(/obj/item/organ/liver)
 	exclude_types = list(/obj/item/organ/liver/cybernetic, /obj/item/organ/liver/cybernetic/upgraded)
 
+/datum/export/orgains/cybernetic
```
Comment (dapnee) [inline code/modules/cargo/exports/organs_robotics.dm]:
```suggestion
/datum/export/organs/cybernetic
```

File: code/modules/cargo/exports/organs_robotics.dm [hunk 6]
```diff
@@ -110,6 +111,17 @@
...
+	cost = 225
+	unit_name = "cybernetic organ"
+	export_types = list(/obj/item/organ/liver/cybernetic, /obj/item/organ/lungs/cybernetic, /obj/item/organ/eyes/robotic, /obj/item/organ/heart/cybernetic)
+	exclude_types = list(/obj/item/organ/lungs/cybernetic/upgraded, /obj/item/organ/liver/cybernetic/upgraded)
+
+/datum/export/orgains/upgraded
```
Comment (dapnee) [inline code/modules/cargo/exports/organs_robotics.dm]:
```suggestion
/datum/export/organs/upgraded
```

File: code/modules/cargo/exports/organs_robotics.dm [hunk 7]
```diff
@@ -110,6 +111,17 @@
...
+	cost = 275
+	unit_name = "upgraded cybernetic organ"
+	export_types = list(/obj/item/organ/lungs/cybernetic/upgraded, /obj/item/organ/liver/cybernetic/upgraded)
+
 /datum/export/orgains/tail //Shhh
```
Comment (dapnee) [inline code/modules/cargo/exports/organs_robotics.dm]:
```suggestion
/datum/export/organs/tail //Shhh
```

Will probably need to check further for more mentions of “orgains” and switch it to the correct spelling of organs.

**Review by dapnee — CHANGES_REQUESTED** [review]
Misspelling and grammar fixes.

**Review by silicons — APPROVED** [review]

**Pull request merged**